### DISCUSSION TITLE
Example usage together with Bevy?

### DISCUSSION THREAD

**Discussion by TheKnarf** [hannobraun/fornjot#2314] [Q&A]
I would love an example on how to use `fornjot` together with [`Bevy`](https://bevyengine.org/). By that I mean that I'd like to create a model with `fornjot` and then show the model in Bevy, maybe even with a simple UI to change some parameters.

**Comment by hannobraun** [comment]
This might be helpful: https://github.com/TotalKrill/bevy_mod_fornjot

Converting this to a discussion, since it doesn't track an improvement to Fornjot.

**Reply by TheKnarf** [reply]
Looks great! Maybe it could be linked to from the Readme and/or website?
